### PR TITLE
Feature two column template

### PR DESCRIPTION
**Pull request by mmenavas**
Please review my 2 column responsive template.

### CODE REVIEW

**Comment by Oh-KPond** [comment]
Is this complete? It looks like you're missing something.

**Comment by Oh-KPond** [comment]
Is this complete? It looks like you're missing something.

**Comment by mmenavas** [comment]
Yes. I totally forgot to add the css code. Thanks for reviewing my PR.

File: html-templates/2-column-responsive.html [hunk 1]
```diff
@@ -1,3 +1,29 @@
+<style>
+	/* 2 equal-width columns Layout Pattern CSS */
+	@media only screen and (max-width: 599px) {
+        td[class="pattern"] table { width: 100%; }
+        td[class="pattern"] img {
+            width: 100%;
+            height: auto !important;
```
Comment (Oh-KPond) [inline html-templates/2-column-responsive.html]: This is important! It says so right it right here.

Comment (Oh-KPond) [inline html-templates/2-column-responsive.html]: See! Important!

**Pull request merged**